include/libdnf5/conf/config_main.hpp

```diff
@@ -244,8 +244,12 @@ class LIBDNF_API ConfigMain : public Config {
     const OptionString & get_username_option() const;
     OptionString & get_password_option();
     const OptionString & get_password_option() const;
+    /// @deprecated It is going to be removed, use get_pkg_gpgcheck_option()
```

Comment:
This reminds me, we should already discuss how to consistently handle the deprecated options in the CLI and API (https://github.com/rpm-software-management/dnf5/issues/865). I’ll add it to the agenda for some of our next dev meetings so we can agree on this together.